packages/contracts/test/asset_proxy/proxies.ts

```diff
@@ -244,9 +234,9 @@ describe('Asset Transfer Proxies', () => {
             });
         });
 
-        it('should have an id of 1', async () => {
+        it('should have an id of 0xf47261b0', async () => {
```

Comment:
We should probably make this a constant -- could use the value from `AssetProxyId` enum.